.github/actions/build-backend/action.yml

```diff
@@ -41,8 +41,7 @@ runs:
 
     - name: Lint
       if: inputs.run-integration-tests == 'true'
-      working-directory: prime-router
-      run: ./gradlew ktlintCheck
+      run: ./gradlew :prime-router:ktlintCheck
```

Comment:
thoughts on adding `ktlint` to the new projects? Or is that for later tickets and this ticket is to just get the barebones up?

Reply:
I was thinking of saving that for a follow up